React-Native/en/1.4.2.md

```diff
@@ -0,0 +1,10 @@
+# [Success Criterion 1.4.2 - Audio Control](https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html)
+
+When playing sound it is imperative that a user can control the audio by pausing and starting it.
+
+In order to do this you can use the [`react-native-video` package with the `audioOnly` property enabled](https://github.com/react-native-video/react-native-video/blob/master/API.md#audioonly). Out of the box `react-native-video` will offer the use all the native controls required
```

Comment:
will offer the use all the native controls required
-->
offers native controls for playing and pausing
or
offers native controls to play and pause audio.